js/components/ReportSummaryForm.vue

```diff
@@ -116,6 +130,10 @@ export default {
       this.bucketBy = null;
     },
 
+    clearErrors() {
+      this.errors = [];
```

Comment:
It's not something you need to solve right now, but I think it's usually better to put errors next to the form fields that triggered them, rather than at the top of the form. It tends to make it easier to tell what needs to be fixed.